ci/util.py

```diff
@@ -113,3 +116,14 @@ def chdir_to_script_directory():
     os.chdir(base)
 
 
+def script_name() -> str:
+    """:returns: script name with leading paths removed"""
+    return os.path.split(sys.argv[0])[1]
+
+
+def config_logging():
```

Comment:
Could we update this with something like:

```

def config_logging():
    logging.config.fileConfig(os.getenv('LOGGING_CONF', 'logging.conf'))
```


```
[loggers]
keys=root,simpleExample

[handlers]
keys=consoleHandler

[formatters]
keys=simpleFormatter

[logger_root]
level=DEBUG
handlers=consoleHandler

[logger_simpleExample]
level=DEBUG
handlers=consoleHandler
qualname=simpleExample
propagate=0

[handler_consoleHandler]
class=StreamHandler
level=DEBUG
formatter=simpleFormatter
args=(sys.stdout,)

[formatter_simpleFormatter]
format=%(asctime)s - %(name)s - %(levelname)s - %(message)s
datefmt=
```

I'm moving away from these hardcoded logging statements.